deltametrics/section.py

```diff
@@ -301,16 +301,24 @@ def __getitem__(self, var):
         """
         if type(self.cube) is cube.DataCube:
             if self.cube._knows_stratigraphy:
-                return DataSectionVariable(_data=self.cube[var][:, self._y, self._x],
+                return DataSectionVariable(_data=self.cube[var].data.values[:,
+                                                                     self._y,
+                                                                     self._x],
```

Comment:
I guess this is some of the stuff that is "clumsy" right now? I agree there may be a better way to do this, but I don't know it for sure.

Reply:
Yes it feels like there should be a better way... maybe we'll happen upon it as we move towards more `xarray` objects and start actually using the coordinates etc.